README.md

```diff
@@ -0,0 +1,50 @@
+Piinguin lambda relay
+=====================
+
+Pushes PII events to Piinguin using the piinguin client. 
+
+This relay is meant to listen to EnrichedEvent formatted messages coming on an AWS Kinesis stream containing all the extracted PII from Snowplow. It then forwards those to a `piinguin-server`. Please read the piinguin project documentation for more information [here][piinguin].
+
+Deployment
+==========
+
+You can always use the bintray version [here][bintray].
+
+If you need to rebuild and deploy yourself then follow those steps:
+
+* `sbt assembly`
+* Copy artifact to S3 (it's too large for the console) by e.g. `aws s3 cp target/scala-2.12/piinguin-relay-assembly-0.1.0-SNAPSHOT.jar s3://somebucket/`
```

Comment:
SNAPSHOT

Reply:
Changed it, however SNAPSHOT makes sense as if you are building it yourself as opposed to using the bintray artifacts it is a 'SNAPSHOT'